tests/Doctrine/Tests/Common/Collections/Expr/CompositeExpressionTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Doctrine\Tests\Common\Collections\Expr;
+
+use Doctrine\Common\Collections\Expr\CompositeExpression;
+use Doctrine\Common\Collections\Expr\ExpressionVisitor;
+use PHPUnit_Framework_TestCase as TestCase;
+
+/**
+ * @author  Tobias Oberrauch <hello@tobiasoberrauch.com>
+ * @covers  \Doctrine\Common\Collections\Expr\CompositeExpression
+ */
+class CompositeExpressionTest extends TestCase
+{
+    /**
+     * @return CompositeExpression
+     */
+    public function testConstructor()
```

Comment:
I don't understand what you did here. you kept the test which is not testing anything (test without assertion), and you removed the test which was covering the case of exceptions (ensuring we don't accept composite without expressions). What we asked is the opposite